plugins/hypervisors/vmware/src/main/java/com/cloud/hypervisor/vmware/manager/VmwareStorageManagerImpl.java

```diff
@@ -1014,16 +1018,19 @@ private void exportVolumeToSecondaryStroage(VirtualMachineMO vmMo, String volume
                 String datastoreVolumePath = getVolumePathInDatastore(dsMo, volumePath + ".vmdk", searchExcludedFolders);
                 workerVm.attachDisk(new String[] {datastoreVolumePath}, morDs);
                 vmMo = workerVm;
```

Comment:
if we move clonedWorkerVMNeeded outside the try workerVm is not needed.

Reply:
Same reason as before, using different instances/variable, it's easier to track is we need to destroy the worker vm after we're done. If we re-use `vmMo`, we might accidently destroy the user-vm.